src/traffic_server/InkIOCoreAPI.cc

```diff
@@ -161,6 +161,33 @@ TSThreadCreate(TSThreadFunc func, void *data)
   return reinterpret_cast<TSThread>(thread);
 }
 
+TSThread
+TSThreadCreateSignal(TSThreadFunc func, void *data, bool term_by_signal)
+{
+  INKThreadInternal *thread;
```

Comment:
Why doesn't this call `TSThreadCreate` instead of duplicating the code? Or move that code to a shared function.
Or `TSThreadCreate` call this with a `false` argument?